docs/book/src/clusterctl/developers.md

```diff
@@ -177,9 +177,7 @@ EOF
 The command for getting the kubeconfig file for connecting to a workload cluster is the following:
 
 ```bash
-kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o jsonpath={.data.value} \
-  | base64 --decode \
-  > ./capi-quickstart.kubeconfig
+clusterctl get kubeconfig capi-quickstart --namespace=default
```

Comment:
is it possible to omit --namespace=default ?

Reply:
Also, should we document that for CAPD on mac os the server address should be changed?

Reply:
I believe it is documented below 🙂